src/selectors/blockchain.js

```diff
@@ -23,3 +28,27 @@ export const getTokenAmount = (state, tokenAddress) => {
 }
 
 export const getTokenSymbol = (state, tokenAddress) => state.blockchain.getIn(['tokenSymbols', tokenAddress])
+
+/**
+ * @param {*} state - redux state
+ * @returns {object} collateralToken - with Symbol, Amount in ETH and Address
+ */
+export const getCollateralTokenAmount = (state) => {
+  const collateralToken = getCollateralToken()
+
+  if (!collateralToken) {
+    return {
+      symbol: 'ETH',
+      amount: weiToEth(getCurrentBalance(state)),
+      address: undefined,
```

Comment:
what about returning an address of deployed etherToken from gnosis.js?
```
address: gnosis.etherToken.address
```

Reply:
That'd be wrapped ether, the header should display just ether for now